### ISSUE TITLE
Accessibility - Accessible Name for Frame/Axe-Core

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#81]
## Improving QDR's Dataverse for Qualitative Data: https://qdr.syr.edu/qdr-blog/improving-qdrs-dataverse-qualitative-data

- [x] Video Frame should have an accessible name and should contain the axe-core script

## Webinar: A Tale of Two Data Projects: https://qdr.syr.edu/qdr-blog/webinar-tale-two-data-projects

- [x] Video Frame should have an accessible name and should contain the axe-core script

## JDSTOR and QDR Partner in Organizing Data Rescue Event: https://qdr.syr.edu/qdr-blog/jstor-and-qdr-partner-organizing-data-rescue-event

- [x] Video Frame should have an accessible name and should contain the axe-core script

## Webinar on Securely Managing Qualitative Data: https://qdr.syr.edu/qdr-blog/webinar-securely-managing-qualitative-data

- [x] Video Frame should have an accessible name and should contain the axe-core script

**Comment by adam3smith** [comment]
All fixed
